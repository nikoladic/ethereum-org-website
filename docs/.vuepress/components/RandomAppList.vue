<template>
  <RandomList :n="7" :items="items"></RandomList>
</template>

<script>
import { translate } from '../theme/utils/translations'
import RandomList from './RandomList.vue'

export default {
  components: { RandomList },

  computed: {
    items() {
      return [
        {
          name: 'Gitcoin',
          url: 'https://gitcoin.co',
          description: translate('dapp-desc-gitcoin', this.$lang)
        },
        {
          name: 'Cent',
          url: 'https://beta.cent.co',
          description: translate('dapp-desc-cent', this.$lang)
        },
        {
          name: 'Gods Unchained',
          url: 'https://godsunchained.com/',
          description: translate('dapp-desc-gods-unchained', this.$lang)
        },
        {
          name: 'DAI',
          url: 'https://makerdao.com/en/',
          description: translate('dapp-desc-makerdao', this.$lang)
        },
        {
          name: 'Decentraland',
          url: 'https://decentraland.org/',
          description: translate('dapp-desc-decentraland', this.$lang)
        },
        {
          name: 'Dharma',
          url: 'https://www.dharma.io/',
          description: translate('dapp-desc-dharma', this.$lang)
        },
        {
          name: 'Augur',
          url: 'https://www.augur.net/',
          description: translate('dapp-desc-augur', this.$lang)
        }
      ]
    }
  }
}
</script>
