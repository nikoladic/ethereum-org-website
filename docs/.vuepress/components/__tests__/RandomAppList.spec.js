import { shallowMount } from '@vue/test-utils'
import RandomAppList from "@/components/RandomAppList"

describe('RandomAppList', () => {
  test('render randon app list', () => {
    const wrapper = shallowMount(RandomAppList)
    expect(wrapper.element).toMatchSnapshot()
  })
})
