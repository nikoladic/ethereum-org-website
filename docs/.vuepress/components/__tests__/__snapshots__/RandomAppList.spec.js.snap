// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RandomAppList render randon app list 1`] = `
<randomlist-stub
  comp="[object Object]"
  items="[object Object],[object Object],[object Object],[object Object]"
  n="4"
/>
`;
